pkg/ansible/proxy/inject_owner.go

```diff
@@ -74,15 +74,15 @@ func (i *injectOwnerReferenceHandler) ServeHTTP(w http.ResponseWriter, req *http
 		k, err := getGVKFromRequestInfo(r, i.restMapper)
 		if err != nil {
 			// break here in case resource doesn't exist in cache
-			log.Info("Cache miss, can not find in rest mapper")
+			log.Info("Cache miss, can not find in rest mapper", "error", err.Error())
```

Comment:
Why we are not using the log.Error here instead of info?

Reply:
Well it might not be an actual error, it could be totally fine, but we're just losing the context for why it didn't work. Not sure if it should show up as an error level log though